### ISSUE TITLE
Spawning of creatures does not respect light level

### ISSUE THREAD

**Issue by Knito58** [AzulConspirator/AzulConspirator-Create-ContraptionCreatures#7]
Is this a bug or a feature? In a forest they will spawn even in well lit areas.

**Comment by SirNaranjas** [comment]
I read the spawn code of this creatures and this seems to be a feature, because the spawn parameters make the creatures only spawn in forest and the of that is make the create most used materials more easy to get without go to mine  in caves. If you see too much loot from the creatures and you know sometehing of edit mods, you can change some loot tables of the creatures to make the loot not too much OP.

**Comment by Knito58** [comment]
Yes that is right. But I would rather change things in game (placing torches) than in config. I like it to have a lot of resources coming to me. In most forests they are welcome. But not just at the entrance to my base.
